schema/anyof.go

```diff
@@ -28,3 +30,23 @@ func (v AnyOf) Validate(value interface{}) (interface{}, error) {
 	// TODO: combine errors.
 	return nil, errors.New("invalid")
 }
+
+// Serialize attempts to serialize the value using the first available
+// FieldSerializer which does not return an error. If no appropriate serializer
+// is found, the function returns the input value.
```

Comment:
Make slightly shorter, and avoid calling a method for a function which (although true) can cause slight mental confusion for some: 

the function returns the input value. -> the input value is returned.